package.json

```diff
@@ -62,6 +62,7 @@
     "@svgr/cli": "^6.2.1",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^12.8.3",
```

Comment:
You didn't change this, but... right now our `@types/react-transition-group` version is `v.4.4.0`, whereas the actually version of `react-transition-group` we install is `v2.9.0`. We should update them both to the latest `v4.4.2` (there's no breaking changes that affect us). May as well do that as part of this change, so find/replace all `react-transition-group` version specs in all package.jsons you can find with the latest version then run `npm install` again to update package lock.